### PR TITLE
En_Snowwd documentation

### PR DESCRIPTION
**Pull request by hack-nuss**

### CODE REVIEW

File: src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c [hunk 1]
```diff
@@ -79,23 +71,27 @@ void EnSnowwd_Destroy(Actor* thisx, PlayState* play) {
...
     if (this->collider.base.acFlags & AC_HIT) {
         this->collider.base.acFlags &= ~AC_HIT;
         Actor_PlaySfx(thisx, NA_SE_IT_REFLECTION_WOOD);
     }
     if (thisx->home.rot.y != 0) {
-        this->unk190 = 0x15;
+        this->timer = 21;
         thisx->home.rot.y = 0;
         if (thisx->home.rot.z == 0) {
             if (SNOWWD_GET_PARAM_F80(&this->actor) < 0x10) {
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c:91]:
```suggestion
            if (SNOWWD_GET_DROP_TABLE(&this->actor) < DROP_TABLE_SIZE) {
```
Personally this makes sense to me. May want to get a second opinion before implementing it though. 
Note the `DROP_TABLE_SIZE` would apply to the `0x10` multiplier below in the `Item_DropCollectibleRandom` call.

**Review by hensldm — COMMENTED** [review]
Nice work :) . Some thoughts.

**Comment by AngheloAlf** [comment]
The review were marked as resolved but there are no changes. @hack-nuss did you forgot to push?

**Comment by hack-nuss** [comment]
I'm on the train and got bad WiFi.

File: src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c [hunk 2]
```diff
@@ -50,26 +50,18 @@ static ColliderCylinderInit sCylinderInit = {
...
-
-Color_RGBA8 D_80AF7AD4 = { 255, 255, 255, 255 };
-
-Color_RGBA8 D_80AF7AD8 = { 200, 200, 220, 0 };
-
 void EnSnowwd_Init(Actor* thisx, PlayState* play) {
     EnSnowwd* this = THIS;
 
-    this->actor.home.rot.z = 0;
+    SNOWWD_DROPPED_COLLECTIBLE(thisx) = 0;
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c:56]:
```suggestion
    SNOWWD_DROPPED_COLLECTIBLE(thisx) = false;
```

File: src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c [hunk 3]
```diff
@@ -78,43 +70,49 @@ void EnSnowwd_Destroy(Actor* thisx, PlayState* play) {
...
     if (thisx->home.rot.y != 0) {
-        this->unk190 = 0x15;
+        this->timer = 21;
         thisx->home.rot.y = 0;
-        if (thisx->home.rot.z == 0) {
-            if (SNOWWD_GET_PARAM_F80(&this->actor) < 0x10) {
-                sp48 = thisx->world.pos;
-                sp48.y += 200.0f;
-                Item_DropCollectibleRandom(play, NULL, &sp48, SNOWWD_GET_PARAM_F80(&this->actor) * 0x10);
+        if (SNOWWD_DROPPED_COLLECTIBLE(&this->actor) == 0) {
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c:90]:
```suggestion
        if (!SNOWWD_DROPPED_COLLECTIBLE(&this->actor)) {
```

File: src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c [hunk 4]
```diff
@@ -78,43 +70,49 @@ void EnSnowwd_Destroy(Actor* thisx, PlayState* play) {
...
+            if (SNOWWD_GET_DROP_TABLE(&this->actor) < DROP_TABLE_SIZE) {
+                pos = thisx->world.pos;
+                pos.y += 200.0f;
+                Item_DropCollectibleRandom(play, NULL, &pos, SNOWWD_GET_DROP_TABLE(&this->actor) * DROP_TABLE_SIZE);
             }
-            thisx->home.rot.z = 1;
+            SNOWWD_DROPPED_COLLECTIBLE(&this->actor) = 1;
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c:96]:
```suggestion
            SNOWWD_DROPPED_COLLECTIBLE(&this->actor) = true;
```

File: src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c [hunk 5]
```diff
@@ -78,43 +70,49 @@ void EnSnowwd_Destroy(Actor* thisx, PlayState* play) {
...
-        this->unk190 = 0x15;
+        this->timer = 21;
         thisx->home.rot.y = 0;
-        if (thisx->home.rot.z == 0) {
-            if (SNOWWD_GET_PARAM_F80(&this->actor) < 0x10) {
-                sp48 = thisx->world.pos;
-                sp48.y += 200.0f;
-                Item_DropCollectibleRandom(play, NULL, &sp48, SNOWWD_GET_PARAM_F80(&this->actor) * 0x10);
+        if (SNOWWD_DROPPED_COLLECTIBLE(&this->actor) == 0) {
+            if (SNOWWD_GET_DROP_TABLE(&this->actor) < DROP_TABLE_SIZE) {
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c:91]:
```suggestion
            if (SNOWWD_GET_DROP_TABLE(&this->actor) < 16) {
```
🤔 Hmm thinking about this again, `DROP_TABLE_SIZE` actually doesn't make sense here, it should refer to the `DROP_TABLE_NUMBER` instead. Probably best to just set it back to 16 for now. (Note the `DROP_TABLE_SIZE` below does still make sense and should be left).

**Review by hensldm — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Review by engineer124 — APPROVED** [review]

**Pull request merged**